src/internal/operations/backup.go

```diff
@@ -335,28 +339,28 @@ func collectMetadata(
 	return dcs, nil
 }
 
-func selectorToOwnersCats(sel selectors.Selector) *kopia.OwnersCats {
+func selectorToReasons(sel selectors.Selector) []kopia.Reason {
 	service := sel.PathService()
-	oc := &kopia.OwnersCats{
-		ResourceOwners: map[string]struct{}{},
-		ServiceCats:    map[string]kopia.ServiceCat{},
-	}
-
-	oc.ResourceOwners[sel.DiscreteOwner] = struct{}{}
+	reasons := []kopia.Reason{}
 
 	pcs, err := sel.PathCategories()
 	if err != nil {
-		return &kopia.OwnersCats{}
+		// This is technically safe, it's just that the resulting backup won't be
+		// usable as a base for future incremental backups.
+		return nil
 	}
 
 	for _, sl := range [][]path.CategoryType{pcs.Includes, pcs.Filters} {
```

Comment:
Cat could be duplicated across Includes and Filters, yeah?  Does this need to dedupe?

Reply:
> Something like backup create --user "*,user1" which would go to selectors for all users and then user1 again. That would eventually cause two backups for user1 to be created

Any case that involves a combined declaration of a wildcard and a static value is already handled.  `*,foo` will get reduced to `*`.

But that assumes a single declaration.  If we get something like:
```go
Include(
  MailFolders(Any()),
  MailFolders("Inbox"),
)
```
then we'll get duplicate collections on the inbox pathing.  IE: the cli is safe from this, but not the sdk.  This could be handled in either selectors or collections, depending on where we want to put the mitigation.